### ISSUE TITLE
Bug when using buildFunction for acsc, asec and acot. Bug where calculation of acot is wrong.

### ISSUE THREAD

**Issue by Saikedo** [jiggzson/nerdamer#455]
Ok, so I made another pull request (#454) and this pull addresses couple of issues.

1) A crash that was caused by buildFunction for acsc and asec has been fixed.

2) Bug, where the calculation of acot for negative numbers was wrong, has been fixed.

3) After step 2, I needed to fix another bug for buidFunction that would cause a crash for acot.

Martin, please let me know if you need an explanation for why I think issue 2 was necessary. There are some conflicting opinions about this and I had to confirm my suspicions with few Math professors before I felt comfortable introducing the fix.

**Comment by jiggzson** [comment]
@Saikedo, I left a comment on the PR but if you want we can continue the discussion here.

**Comment by Saikedo** [comment]
@jiggzson I left a comment on the RP so let`s just continue there I guess :)

**Comment by jiggzson** [comment]
See #454

**Comment by Saikedo** [comment]
@jiggzson Was there a decision made in #454 ? I checked the commit request and could not find anything new there.

**Comment by jiggzson** [comment]
Yes. I'm going to be merging this PR. Can you pull the recent changes? Saves me some time.

**Comment by Saikedo** [comment]
@jiggzson Will handle that once I get home from work. I have another push to make for Nerdamer anyway so I will handle both of those this evening.
